security-framework/src/trust_settings.rs

```diff
@@ -111,6 +113,19 @@ impl TrustSettings {
 
         Ok(TrustSettingsIter { index: 0, array })
     }
+    ///set trust settings to ""always trust this root certificate regardless of use.".
+    ///It is not possible to modify per-user trust settings when not running in a GUI environment
+    pub fn set_trust_settings(&self, cert: &SecCertificate) -> Result<()> {
+        let domain = self.domain;
+        let trust_settings: CFTypeRef = ptr::null_mut();
+        cvt(unsafe {
+            SecTrustSettingsSetTrustSettings(
+                cert.as_CFTypeRef() as *mut _,
+                domain.into(),
+                trust_settings,
```

Comment:
should it take a dictionary? Would someone need to set these settings?

If not, maybe rename the method to indicate it uses empty settings.